tests/hikari/impl/test_rest.py

```diff
@@ -2648,6 +2648,39 @@ async def test_create_guild_voice_channel(self, rest_client):
             reason="because we need one",
         )
 
+    async def test_create_guild_stage_channel(self, rest_client):
+        guild = StubModel(123)
+        channel = mock.Mock(channels.GuildStageChannel)
+        category_channel = StubModel(789)
+        overwrite1 = StubModel(987)
+        overwrite2 = StubModel(654)
+        rest_client._create_guild_channel = mock.AsyncMock(return_value=channel)
+
+        returned = await rest_client.create_guild_stage_channel(
+            guild,
+            "general",
+            position=1,
+            user_limit=60,
+            bitrate=64,
+            permission_overwrites=[overwrite1, overwrite2],
+            category=category_channel,
+            region="Doge Moon",
+            reason="When doge == 1$",
+        )
+        assert returned == channel
```

Comment:
couldn't `is` be used here to check?

Reply:
oh didn't notice that, might as well just leave that style then ig

Reply:
These should def be using `is`. I will have to fix the other tests